core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/catalog/service/CategorySiteMapGenerator.java

```diff
@@ -61,13 +61,14 @@ public boolean canHandleSiteMapConfiguration(SiteMapGeneratorConfiguration siteM
     @Override
     public void addSiteMapEntries(SiteMapGeneratorConfiguration smgc, SiteMapBuilder siteMapBuilder) {
 
-        CategorySiteMapGeneratorConfiguration categorySMGC = (CategorySiteMapGeneratorConfiguration) smgc;
+        if (smgc instanceof CategorySiteMapGeneratorConfiguration) {
```

Comment:
Change this to `CategorySiteMapGeneratorConfiguration.class.isAssignableFrom(smgc.getClass())`